src/main/java/controller/LadderGameController.java

```diff
@@ -0,0 +1,86 @@
+package controller;
+
+import domain.*;
+import util.BooleanGenerator;
+import view.input.InputView;
+import view.output.OutputView;
+
+public class LadderGameController {
+
+    private static final String GET_RESULT_ALL = "all";
```

Comment:
GET은 붙이지 않아도 되지 않을까요?
```suggestion
    private static final String RESULT_ALL = "all";
```